src/realm-settings/LocalizationTab.tsx

```diff
@@ -87,6 +103,34 @@ export const LocalizationTab = ({
     setAddMessageBundleModalOpen(!addMessageBundleModalOpen);
   };
 
+  const options = [
+    <SelectGroup label={t("defaultLocale")} key="group1">
+      {watchSupportedLocales
+        .filter((item) => item === realm?.defaultLocale)
+        .map((locale: string, idx: number) => (
+          <SelectOption key={`supported-locale-${idx}`} value={locale}>
+            {t(`allSupportedLocales.${locale}`)}
+          </SelectOption>
+        ))}
+    </SelectGroup>,
+    <Divider key="divider" />,
+    <SelectGroup label={t("supportedLocales")} key="group2">
+      {watchSupportedLocales
+        .filter((item) => item !== realm?.defaultLocale)
+        .map((locale: string, idx: number) => (
+          <SelectOption key={`supported-locale-${idx}`} value={locale}>
```

Comment:
Is `locale` not already unique in this context? React [recommends](https://reactjs.org/docs/lists-and-keys.html#keys) using the `index` as a key only as a last resort.
```suggestion
          <SelectOption key={locale} value={locale}>
```